public/less/_app.variables.less

```diff
@@ -144,7 +144,7 @@
 // -------------------------------------------- MAIN CONTENT ---------------------------------------------
 @main-font-color: @color-biscay-blue-s41;
 @main-navbar-default-bg-color: @light-text;
-@main-width: 100% ~"-" @sidebar-width;
+@main-width: ~"calc(100% - 30rem)";
```

Comment:
We can't do this because of well known issue with calc and less pre-processor, where such expression will be parsed as calc(100% - 30%) if we not make string from this;